quickevent/app/quickevent/plugins/CardReader/src/cardreaderplugin.cpp

```diff
@@ -4,6 +4,7 @@
 #include "cardreaderwidget.h"
 #include "cardreadersettingspage.h"
 #include "services/racomclient.h"
+#include "services/webapi.h"
```

Comment:
`webapi` is not good name for service, please use `qropunch` or some other name more descriptive.

Reply:
Ok, will do.